galley/pkg/config/analysis/analyzers/authz/authorizationpolicies.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 Istio Authors
```

Comment:
2020, I'm actually not sure if we still include year in the copyright, I remember seeing @howardjohn having some PR to remove this.

Reply:
Oopsie. I see that being removed. Approached.